### ISSUE TITLE
Routing is slow

### ISSUE THREAD

**Issue by xivk** [oSoc18/bike4brussels#4]
We need to make the routing faster, it takes > 4secs for the routing server to respond. This is ok for testing but not in production.

**Comment by Fizz55** [comment]
I ran IDP with the --contract bicycle option, but that does not seem to have that much of an impact. For longer routes it might be noticably faster, but the shorter ones still take around 3 to 4 seconds.

**Comment by xivk** [comment]
Has this been confirmed to be the instructions?

**Comment by Fizz55** [comment]
Not really, because setting instructions to false in the request didn't seem to disable them
